src/foam/u2/Progressable.js

```diff
@@ -0,0 +1,21 @@
+/**
+* @license
+* Copyright 2022 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.INTERFACE({
+  package: 'foam.u2',
+  name: 'Progressable',
+  documentation: '',
+  properties: [
+    {
```

Comment:
Wouldn't it be easier to just have one property which you could then just bind to data$? Or are you going to display something like "n of m"?

Reply:
The depending on the kind of progress view we might want to display n of m or just show an abstract progress, I was thinking of defaulting the max to 100 so that you could pass a single prop as a percentage value, not sure if that helps a lot

Reply:
can also display progress as segments if both values exist

Reply:
What if there was a `PortionHolder` with properties `numerator` and `denominator`, and that is the object inside `data`?

Reply:
Isnt that the same thing as two props